src/highdicom/sr/content.py

```diff
@@ -500,6 +514,228 @@ def __init__(
+                [xc, yc],
+                [xc + radius, yc]
+            ]
+        )
+
+        return cls(
+            graphic_type=GraphicTypeValues.CIRCLE,
+            graphic_data=graphic_data,
+            source_image=source_image,
+            pixel_origin_interpretation=pixel_origin_interpretation
+        )
+
+    @classmethod
+    def from_box(
+        cls,
+        left: float,
+        right: float,
+        top: float,
+        bottom: float,
+        source_image: SourceImageForRegion,
+        pixel_origin_interpretation: Optional[
+            Union[PixelOriginInterpretationValues, str]
+        ] = None
+    ) -> 'ImageRegion':
+        """Construct an ImageRegion representing an axis-aligned box.
+
+        Note that there is no specialized way to represent a box using graphic
+        data. This is a convenience wrapper that correctly represents an
+        axis-aligned box as a special case of a closed polyline.
+
+        Parameters
+        ----------
+        left: float
+            Zero-based x-coordinate (column index) of the left side of the
+            box, in pixel units from the left of the image.
+        right: float
+            Zero-based x-coordinate (column index) of the right side of the
+            box, in pixel units from the left of the image.
+        top: float
+            Zero-based y-coordinate (row index) of the top of the
+            box, in pixel units from the top of the image.
+        bottom: float
+            Zero-based y-coordinate (row index) of the bottom of the
+            box, in pixel units from the top of the image.
+
+        Returns
+        -------
+        ImageRegion
+            Image Region describing the specified box using the GraphicType
+            ``POLYLINE``. The points are traversed clockwise from the top left
+            and the first and last points of the box are the same to encode a
+            closed box.
+
+        Raises
+        ------
+        ValueError
+            If any of the parameters are negative.
+
+        """
+        if left < 0.0 or top < 0.0:
+            raise ValueError("Image coordinates may not be negative.")
+
+        if right < left:
+            raise ValueError("Right must be greater than left.")
+        if bottom < top:
+            raise ValueError("Bottom must be greater than top.")
+
+        # Points in clockwise order from top left
+        graphic_data = np.array(
+            [
+                [left, top],
+                [right, top],
+                [right, bottom],
+                [left, bottom],
+                [left, top],  # close the polygon
+            ]
+        )
+
+        return cls(
+            graphic_type=GraphicTypeValues.POLYLINE,
+            graphic_data=graphic_data,
+            source_image=source_image,
+            pixel_origin_interpretation=pixel_origin_interpretation
+        )
+
+    @classmethod
+    def from_ellipse(
+        cls,
+        xc: float,
+        yc: float,
```

Comment:
We should avoid `x` and `y` here and use `row` and `col` instead, because the axis of the coordinate system (pixel matrix) are Rows and Columns.